hebiros_advanced_examples/launch/omnibase/omnibase_odometry_navigation.launch

```diff
@@ -0,0 +1,10 @@
+<launch>
+    <node name="hebiros_node" pkg="hebiros" type="hebiros_node" respawn="false" output="screen"/>
+
+    <node pkg="hebiros_advanced_examples" type="example_omnibase_odometry" name="example_omnibase_odometry" />
+
+    <node pkg="hebiros_advanced_examples" type="example_omnibase_free_roam" name="example_omnibase_free_roam" />
+
+    <node pkg="hebiros_advanced_examples" type="key_read.py" name="key_read" />
+
+  </launch>
```

Comment:
How is this different from https://github.com/HebiRobotics/HEBI-ROS/blob/develop/hebiros_advanced_examples/launch/omnibase/example_omnibase_odometry.launch ?